pages/cisco_ios/enable.md

```diff
@@ -0,0 +1,8 @@
+# enable
+
+> Enter privileged execution mode.
+> More information: <https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ios_shl/command/ios-shell-cr-book.pdf>.
```

Comment:
```suggestion
> More information: <https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ios_shl/command/ios-shell-cr-book/ios-shell-cr-a1.html>.
```
The current version is fine too, but should we add the direct HTML link here instead of PDF (since this new link is archivable in places like Internet archive, etc; for future reference if the docs are no longer available).